### PR TITLE
add 3.4.1 release notes

### PR DESCRIPTION
**Pull request by zhuxiujuan28**
add 3.4.1 release notes

### CODE REVIEW

**Comment by zhuxiujuan28** [comment]
Pls help review and merge this PR.
/assign @faweizhao26

File: content/en/docs/v3.4/release/release-v341.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Add pagination for listing repository tags.
+
+
+## Monitoring
+
+### Bug Fixes
+
+- Fix the issue that the Monitoring Target field is displayed blank.
+
+For more information about issues and contributors of KubeSphere 3.4.1, see [GitHub](https://github.com/kubesphere/kubesphere/blob/master/CHANGELOG/CHANGELOG-3.4.1.md).
```
Comment (faweizhao26) [inline content/en/docs/v3.4/release/release-v341.md:96]: We haven't add the changelog of KubeSphere 3.4.1. Can you add it before i merge this pr?

Comment (zhuxiujuan28) [reply content/en/docs/v3.4/release/release-v341.md:96]: waiting for its merge https://github.com/kubesphere/kubesphere/pull/5987

**Pull request merged**